lib/tasks/default_collection_type.rake

```diff
@@ -0,0 +1,13 @@
+namespace :hyrax do
+  namespace :default_collection_type do
+    desc "Create the Default Collection Type"
+    task create: :environment do
+      id = Hyrax::CollectionType.find_or_create_default_collection_type
+      if Hyrax::CollectionType.exists?(machine_id: id)
+        puts "Default collection type is #{id}"
```

Comment:
With the change to have find_or_create_default_collection_type to return an instance of the collection_type.  This validation process will need to be updated.